release/models/aft/openconfig-aft-common.yang

```diff
@@ -579,6 +598,15 @@ submodule openconfig-aft-common {
         also be updated.";
     }
 
+    leaf next-hop-group-name {
+      type string;
+      description
+       "Where applicable, this leaf is a unique identifier string for the
+        next-hop-group. This leaf's type is able to accommodate more
+        permutations of identifiers that vendors may have and want to
+        expose.";
```

Comment:
I'm not quite sure how to parse this -- I understand:

* this is an arbitrary name for the group that is implementation specific

but, it's not clear what "permutation of identifiers" means. What kind of different types of identifiers are we trying to expose?

If it's something that has an enumerated set of values, rather than modelling like this, should we rather model this as an `identity` where each vendor can extend the base identity with enumerated values that have clear definitions?

Reply:
My bad. "permutation of identifiers" made sense in my head.

Its like you said an arbitrary name for the group, that is supported by vendors

Reply:
Looks like this is no longer present in the PR, marking resolved.